### PR TITLE
Adding simple perf benchmark for python clients

### PR DESCRIPTION
**Pull request by tanmayv25**
See the dashboard [here](https://gpuwa.nvidia.com/kibana/goto/fa9990c1e9a94d359d73a21c2c77f64f).
@CoderHam  I have added the ticket to include the pyclients perf in weekly reports.
https://jirasw.nvidia.com/browse/DLIS-2186

### CODE REVIEW

**Review by Tabrizian — APPROVED** [review]

File: qa/L0_perf_pyclients/test.sh [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+MODEL_NAME="custom_zero_1_int32"
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+# Collect HTTP data
```
Comment (CoderHam) [inline qa/L0_perf_pyclients/test.sh:64]: Why not have a for loop that avoids a lot of this duplication and covers http and grpc both?

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: qa/L0_perf_pyclients/test.sh (hunk 1 above)
Comment (tanmayv25) [reply qa/L0_perf_pyclients/test.sh]: Done

**Review by CoderHam — APPROVED** [review]

**Comment by CoderHam** [comment]
Please open the gitlab MR for the corresponding changes as well

**Comment by tanmayv25** [comment]
> Please open the gitlab MR for the corresponding changes as well

Done.. MR #272

**Pull request merged**